index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="normalize.css" type="text/css" media="screen">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <title>Jack Wood-Pearce</title>
+</head>
+<body>
+<header id="home">
+    <a href="#home"><h1>Jack Wood-Pearce</h1></a>
+    <nav>
+        <a href="#portfolio"><div class="nav-1">PORTFOLIO</div></a>
+        <a href="#about"><div class="nav-2">ABOUT</div></a>
+        <a href="#contact"><div class="nav-3">CONTACT</div></a>
+    </nav>
+    <div class="phone">
+        <h3>07777777777</h3>
+        <h3>jackwoodpearce@gmail.com</h3>
+    </div>
+    <a href="#home" class="side-tab"><img src="assets/uparrow.png"></a>
+</header>
+<section>
+    <video autoplay muted loop class="cloud-video">
+        <source src="assets/cloud.mp4" type="video/mp4">
+    </video>
+    <div id="about" class="about">
```

Comment:
Don't know if there's a best practice thing about IDs and classes having the same name?

Reply:
cool changing this